next/src/errors/messages.ts

```diff
@@ -10,13 +10,13 @@ export function getErrorMessage(
   validation: SchemaValidationErrorType,
   customErrorMessage?: string,
 ): string {
-  const presentation = schema['x-jsf-presentation']
+  const presentation = getUiPresentation(schema)
```

Comment:
@lukad @dragidavid @antoniocapelo @eng-almeida from now on, let's remember to use this helper rather than typing directly `x-jsf-presentation`. I wish we had a linter against it... maybe one day ;)